pyresample/test/test_geometry.py

```diff
@@ -1639,11 +1633,11 @@ def suite():
     """
     loader = unittest.TestLoader()
     mysuite = unittest.TestSuite()
-    mysuite.addTest(loader.loadTestsFromTestCase(Test))
+    # mysuite.addTest(loader.loadTestsFromTestCase(Test))
     mysuite.addTest(loader.loadTestsFromTestCase(TestStackedAreaDefinition))
-    mysuite.addTest(loader.loadTestsFromTestCase(TestDynamicAreaDefinition))
-    mysuite.addTest(loader.loadTestsFromTestCase(TestSwathDefinition))
-    mysuite.addTest(loader.loadTestsFromTestCase(TestCrop))
+    # mysuite.addTest(loader.loadTestsFromTestCase(TestDynamicAreaDefinition))
+    # mysuite.addTest(loader.loadTestsFromTestCase(TestSwathDefinition))
+    # mysuite.addTest(loader.loadTestsFromTestCase(TestCrop))
```

Comment:
Why are all the tests commented out ?

Reply:
That was just negligence on my part. I was trying to make certain tests specifically run and didn't uncomment them it looks like! I'll fix this in a few hours.

Reply:
@wroberts4 Let's talk when you get in next. Typically there are ways to specify the exact tests you want to run, even from pycharm.